submission/submission-webapp/src/main/resources/emails/successful-submission.html

```diff
@@ -0,0 +1,43 @@
+<html>
+<body>
+<p>Dear ${USERNAME}<p/>
+<p>
+  Thank you for submitting your data to BioStudies. Your submission "${TITLE}" has been assigned BioStudies accession
+  number ${ACC_NO}.
+</p>
+<p>
+  You'll be able to see it at <a href="https://www.ebi.ac.uk/biostudies/studies/${ACC_NO}">
+  https://www.ebi.ac.uk/biostudies/studies/${ACC_NO}</a> in the next 24 hours.
+</p>
+<div style="display: ${IS_PRIVATE};">
+  <p>
+    The study release date is not set so it's not publicly available. You will be able to see it only by logging in or
+    by accessing it through this link: <a href="https://www.ebi.ac.uk/biostudies/studies/${ACC_NO}/${SECRET_KEY}">
+    https://www.ebi.ac.uk/biostudies/studies/${ACC_NO}/${SECRET_KEY}</a>.
+  </p>
+</div>
+<div style="display: ${IS_PRIVATE_RELEASE_DATE};">

```

Comment:
I think we should not use display property to remove this, as HTML still will be available in user email. We need to use the template engine conditional.

I notice that we do not have any template engine yet in the project, so to solve this we can use freemaker or any other popular template engine for Java (is not that we have super complex requirements here) and change current HTML to be templates of that type. See for example  [Seding Email with Freemarker HTML](https://memorynotfound.com/spring-mail-sending-email-freemarker-html-template-example/)